tests/test_FFBbasis2D.py

```diff
@@ -138,13 +130,58 @@ def testRotate(self):
         y4 = basis.evaluate(v4)
 
         # Rotate 90
-        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-4))
+        self.assertTrue(np.allclose(y1[0], y2[0], atol=1e-5))
+
+        # 2*pi Identity
+        self.assertTrue(np.allclose(x1[0], y3[0], atol=1e-5))
+
+        # Refl (flipped using flipud)
+        self.assertTrue(np.allclose(np.flipud(x1[0]), y4[0], atol=1e-5))
+
+    def testRotateComplex(self):
+        # Now low res (8x8) had problems;
+        #  better with odd (7x7), but still not good.
+        # We'll use a higher res test image.
+        # fh = np.load(os.path.join(DATA_DIR, 'ffbbasis2d_xcoeff_in_8_8.npy'))[:7,:7]
+        # Use a real data volume to generate a clean test image.
+        v = Volume(
+            np.load(os.path.join(DATA_DIR, "clean70SRibosome_vol.npy")).astype(
+                self.dtype
+            )
+        )
+        src = Simulation(L=v.resolution, n=1, vols=v, dtype=v.dtype)
+        # Extract, this is the original image to transform.
+        x1 = src.images[0]
+
+        # __rotate 90 degrees in cartesian coordinates.
+        x2 = Image(np.rot90(x1.asnumpy(), axes=(1, 2)))
+
+        # Express in an FB basis
+        basis = FFBBasis2D((x1.res,) * 2, dtype=x1.dtype)
```

Comment:
we should be able to just instantiate with an int so just `x1.res`

Reply:
Oh cool, I guess we added that feature at some point. Thanks. Changed.